petals-esb-deployer-api/src/main/resources/component-repository.xsd

```diff
@@ -41,6 +48,14 @@
                </documentation>
             </annotation>
          </element>
+         <element name="shared-library-reference" type="cr:sharedLibraryReference" minOccurs="0"
+            maxOccurs="unbounded">
+            <annotation>
+               <documentation>
+                  The optional list of references to shared libraries required by the component.
```

Comment:
It's not a **list** but **an** optional reference to a shared library required by the component